identity/app/controllers/editprofile/editprofile.scala

```diff
@@ -7,7 +7,6 @@ package object editprofile {
   object PublicEditProfilePage extends IdentityPage("/public/edit", "Edit Public Profile")
   object AccountEditProfilePage extends IdentityPage("/account/edit", "Edit Account Details")
   object EmailPrefsProfilePage extends IdentityPage("/email-prefs", "Emails")
-  object MembershipEditProfilePage extends IdentityPage("/membership/edit", "Membership")
```

Comment:
can more code from the old page be removed in a future PR? There should be a controller and a view around? perhaps inside `displayForm`?

Reply:
yes absolutely! once its been working smoothly on the new platform for a few weeks I plan to do a nice spring clean of any old membership tab code in frontend codebase